addons/account/models/company.py

```diff
@@ -205,6 +206,16 @@ def _compute_account_enabled_tax_country_ids(self):
             foreign_vat_fpos = self.env['account.fiscal.position'].search([('company_id', '=', record.id), ('foreign_vat', '!=', False)])
             record.account_enabled_tax_country_ids = foreign_vat_fpos.country_id + record.account_fiscal_country_id
 
+    def _compute_account_onboarding_create_invoice_state(self):
+        for record in self:
+            if not self.env.company.account_onboarding_create_invoice_state_flag:
+                if self.env['account.move'].search_count([('company_id', '=', record.id), ('move_type', '=', 'out_invoice')], limit=1) > 0:
```

Comment:
`search_count(limit=1) > 0`
:exploding_head: 

You have 2 options:
* `search(limit=1)`
* `search_count` (no limit)

And the `> 0` is useless

Not that it really matter because the on boarding won't be active for big db where it might be important. But it is a bit more readable.